lib/tenants/src/dependency.py

```diff
@@ -87,7 +97,8 @@ def decode_hs256(self, token: str) -> Dict[str, Any]:
                 status_code=status.HTTP_401_UNAUTHORIZED,
                 detail="Token is expired!",
             )
-        except (jwt.PyJWTError, jwt.exceptions.DecodeError):
+        except (jwt.PyJWTError, jwt.exceptions.DecodeError) as err:
+            logger.exception(err)
```

Comment:
Exception logger will contain whole error traceback with messages, basically duplicating. We should probably specify own message here.
```suggestion
            logger.exception("Cannot decode invalid token")
```

Reply:
> Exception logger will contain whole error traceback with messages
Yeah, that what `logger.exception` does :) 
> basically duplicating
Check next lines, it raises new `HTTPException` without `from`. So, there is no any chance to get stack trace down stream. Therefore I added explicit logging here